### PR TITLE
Return collection version 'namespace' as a dict (Fixes #5459)

### PR DESCRIPTION
**Pull request by alikins**
https://pulp.plan.io/issues/5459
closes #5459

### CODE REVIEW

**Comment by bmbouter** [comment]
@alikins This looks like it's on the right track. It will need a CHANGES changelog entry though for it to pass the CI.

File: pulp_ansible/app/galaxy/v3/serializers.py [hunk 1]
```diff
@@ -131,6 +131,13 @@ class Meta:
         )
 
 
+class CollectionNamespaceSerializer(serializers.Serializer):
+    """
+    A serializer for a Collection Version namespace field.
+    """
+    name = serializers.CharField(max_length=32, source="namespace")
```
Comment (cutwater) [inline pulp_ansible/app/galaxy/v3/serializers.py]: For serializer that doesn't actually validate data `max_length` is not necessary.

File: pulp_ansible/app/galaxy/v3/serializers.py [hunk 2]
```diff
@@ -141,6 +148,7 @@ class CollectionVersionSerializer(CollectionVersionListSerializer):
     download_url = serializers.SerializerMethodField()
 
     metadata = CollectionMetadataSerializer(source="*")
+    namespace = CollectionNamespaceSerializer(source="*")
```
Comment (cutwater) [inline pulp_ansible/app/galaxy/v3/serializers.py]: Minor: SerializerMethodField would've been more than enough.

File: pulp_ansible/app/galaxy/v3/serializers.py (hunk 1 above)
Comment (alikins) [reply pulp_ansible/app/galaxy/v3/serializers.py]: updated

File: pulp_ansible/app/galaxy/v3/serializers.py (hunk 2 above)
Comment (alikins) [reply pulp_ansible/app/galaxy/v3/serializers.py]: https://github.com/pulp/pulp_ansible/compare/master...alikins:collection_ver_5459_method is also available, but Serializer class seems more pulp style.

**Pull request merged**